installers/msi-language/ActiveState/Rollbar.cs

```diff
@@ -73,13 +73,35 @@ private static void SetRollbarReportingUser(string id, string userName)
             person.UserName = userName;
             RollbarLocator.RollbarInstance.Config.Person = person;
         }
+    }
+}
+
+public class RollbarReport
+{
+    private static RollbarReport _instance;
+    private static object syncLock = new object();
+
+    public static readonly TimeSpan RollbarTimeout = TimeSpan.FromSeconds(10);
+
+    protected RollbarReport() { }
 
-        public static void Report(string message, IDictionary<string, object> customFields = null )
+    public static void Critical(string message, IDictionary<string, object> customFields = null)
+    {
+        lock (syncLock)
         {
-            RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Error(new GenericException(message), customFields);
+            if (_instance == null)
+            {
+                _instance = new RollbarReport();
+                RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Critical(new GenericException(message), customFields);
+            }
         }
+    }
 
+    public static void Error(string message, IDictionary<string, object> customFields = null)
+    {
+        RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Error(new GenericException(message), customFields);
```

Comment:
See AC:
>  When a critical error is encountered no further reports are send 

This goes for all errors.

Reply:
Ahh okay, for some reason I interpreted this as no further _critical_ errors.